### PR TITLE
Flag voice messages as such in protos.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @michaelkirk

### CODE REVIEW

**Comment by charlesmchen** [comment]
See: https://github.com/WhisperSystems/Signal-iOS/pull/2083

File: src/Messages/OWSSignalServiceProtos.pb.m [hunk 1]
```diff
@@ -5865,6 +5916,26 @@ - (OWSSignalServiceProtosAttachmentPointerBuilder*) clearFileName {
   resultAttachmentPointer.fileName = @"";
   return self;
 }
+- (BOOL)hasFlags
```
Comment (michaelkirk) [inline src/Messages/OWSSignalServiceProtos.pb.m:5919]: For some reason editing generated code, even if it's just clang-format, gives me the howling fantods. But I'm willing to accept that this is a reasonable thing to do.

**Review by michaelkirk — APPROVED** [review]
looks good!

File: src/Messages/OWSSignalServiceProtos.pb.m (hunk 1 above)
Comment (charlesmchen) [reply src/Messages/OWSSignalServiceProtos.pb.m:5919]: Fixed.

**Pull request merged**